tools/eslint-plugin-azure-sdk/src/rules/ts-apiextractor-json-types.ts

```diff
@@ -0,0 +1,72 @@
+/**
```

Comment:
I'm not sure about the `azure-sdk-tools` repo, but I think our standard copyright header is

```
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.
```

Reply:
We should probably run a subset of this linter's rules on itself, since this is the kind of thing our linter config can fix 😄

Reply:
The copyright header rule is pretty buggy right now so I am going to defer that to another PR in which I'll fix the linter and also add copyright headers to all the files using the fixer. It will add every single file in the repo to this diff if I do it here & the current fixer doesn't put the notice at the very top of the file.